File: eRents.Domain/Entities/City.cs
using System;
using System.Collections.Generic;

namespace eRents.Domain.Entities;

public partial class City
{
	public int CityId { get; set; }

	public string CityName { get; set; } = null!;

	public int StateId { get; set; }

	public virtual ICollection<Property> Properties { get; set; } = new List<Property>();

	public virtual State State { get; set; } = null!;
}

File: eRents.UI/e_rents_mobile/lib/core/models/city.dart
class City {
  final int cityId;
  final String cityName;
  final int stateId;

  City({required this.cityId, required this.cityName, required this.stateId});

  factory City.fromJson(Map<String, dynamic> json) {
    return City(
      cityId: json['cityId'],
      cityName: json['cityName'],
      stateId: json['stateId'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'cityId': cityId,
      'cityName': cityName,
      'stateId': stateId,
    };
  }
}
